src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheckTest.java

```diff
@@ -89,6 +90,7 @@ public void testCommentIsAtTheEndOfBlock() throws Exception {
             "557:1: " + getCheckMessage(MSG_KEY_SINGLE, 555, 0, 8),
             "562:1: " + getCheckMessage(MSG_KEY_SINGLE, 561, 0, 8),
             "577:1: " + getCheckMessage(MSG_KEY_SINGLE, 574, 0, 8),
+            "589:13: " + getCheckMessage(MSG_KEY_SINGLE, 587, 12, 8),
```

Comment:
> There is a difference between actual and ignore list. There are mutations in your code which are surviving. If you don't know about pit testing, you can read about it from [here](https://pitest.org/). You can run pitests locally also, in your case by-
> `./.ci/pitest.sh pitest-indentation`
> The report will be located in target folder. A mutation generally survives when you have-
> 
> 1. Introduced some duplicated logic in your new code, and/ or
> 2. Modified/ removed some of the existing UT's
> 3. Missing test cases i.e. the test cases you have written are not for all the conditions or piece of code.

Okay, I think its occurring because of the point 2 since I  changed the violation list. 
@Vyom-Yadav how can I resolve it for point 2

Reply:
Have you generated the pit report? working on assumption is not a good way to proceed, just run the command I gave you earlier and see which mutations are not being killed, they will be highlighted in red.

Reply:
Okay, sure.

Reply:
Report is not getting generated. The bash is opening and closing showing some `ERROR`

Reply:
![image](https://user-images.githubusercontent.com/42171790/116663594-f4ba7580-a9b4-11eb-8eff-9d6e6a16dd09.png)

Reply:
Do you have any compiler for shell scripts? Share the whole log

Reply:
Share the whole log.

Reply:
Its resolved now.

Reply:
Thanks a lot @Vyom-Yadav for helping out.

Not getting any more issues now.